src/SDKs/Reservations/Reservations.Tests/Reservations.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Reservations.Tests</PackageId>
+    <Description>Reservations.Tests Class Library</Description>
+    <Authors>Microsoft Corporation</Authors>
+    <AssemblyName>Reservations.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <OutputType>Exe</OutputType>
```

Comment:
@corquiri these tests are not an app/exe. If you model the test project looking at other test projects that will help to make it consistent across.

Reply:
removed